components/menteeDetails/BubbleList.js

```diff
@@ -4,20 +4,10 @@ import BubbleRect from './BubbleRect.js';
 
 class BubbleList extends Component {
   render() {
-    const styles = StyleSheet.create({
-      bubbleContainer: {
-        width: 150
-      },
-      bubbleList: {
-        flexDirection: 'row',
-        flexWrap: 'wrap'
-      }
-    });
-
     const items = [];
-    this.props.items.forEach((item, idx) => {
+    this.props.items.forEach((item) => {
       items.push(
-        <View key={idx} style={styles.bubbleContainer}>
+        <View key={item.name} style={styles.bubbleContainer}>
```

Comment:
Can we enforce that the item have an id field (item.uniqueId) so that we don't have to enforce uniqueness on the name field?

Reply:
we can't enforce it using eslint but we can just do that ourselves in the code by assigning `item.uniqueId` if it exists

Reply:
Yeah, if we adjust this to be item.uniqueId here, that should be good